src/prices/prices.utils.ts

```diff
@@ -52,6 +56,31 @@ export async function getPrice(address: string, currency?: Currency): Promise<To
   }
 }
 
+export async function fetchPrices(chain: Chain, addresses: string[]): Promise<PriceData> {
```

Comment:
initial coingecko batching implementation (not used, will add tests in a smaller follow up) + initial hook ups / use